dali/datest/datest.cpp

```diff
@@ -1730,7 +1730,7 @@ void TestExternal()
             Owned<IFileIO> fileIO = ifile->open(IFOread);
             assertex(fileIO);
 
-            sz = ifile->size();
+            sz = (size32_t)ifile->size();
```

Comment:
This looks like it's in dead code anyway (I hope!) If not it would be troublesome for various reasons...